# Request
sensor_msgs/Image image
geometry_msgs/Point32[] queries
int32[] removed_indices

---
# Response
sensor_msgs/PointCloud forward_points